content/en/docs/chart_best_practices/_index.md

```diff
@@ -5,11 +5,9 @@ weight: 4
 aliases: ["/docs/topics/chart_best_practices/"]
 ---
 
-# The Chart Best Practices Guide
```

Comment:
Removing these top level headings is causing them to be removed from the docs. For example, see https://deploy-preview-732--helm-merge.netlify.app/docs/chart_best_practices/. This applies elsewhere, too.